apps/mull-ui/src/app/components/event-card/event-card.tsx

```diff
@@ -24,7 +25,7 @@ export const EventCard = ({ event, style = {} }: EventCardProps) => {
 
   const { day, month, time } = formatDate(event.startDate);
   return (
-    <div className="event-card-container" style={style}>
+    <div className="event-card-container" onClick={onClick} style={style}>
```

Comment:
You can add the `button` class here so that the cursor becomes a pointer like you wanted